### PR TITLE
feat(meilisearch): add basic set of rules

### PR DESCRIPTION
**Pull request by nohant**
this should add a basic set of rules for meilisearch, i would like to expand this in the future when we will add more in our production cluster.

Documentation for metrics is here: https://www.meilisearch.com/docs/learn/experimental/metrics
im not supergood at this but i want to give something back to the community! :D

### CODE REVIEW

File: _data/rules.yml [hunk 1]
```diff
@@ -842,6 +842,21 @@ groups:
...
 
+      - name: Meilisearch
+        exporters:
+          - name: Embedded exporter
+            slug: embedded-exporter
+            doc_url: https://github.com/orgs/meilisearch/discussions/625
+            rules:                
+              - name: Meilisearch index is empty
+                description: Meilisearch instance is down
+                query: 'meilisearch_index_docs_count{index="$Index"} == 0'
```
Comment (samber) [inline _data/rules.yml:853]: i think `{index="$Index"}` is not expected here

Comment (nohant) [reply _data/rules.yml:853]: the filter in the dashboard for grafana we use is configured this way: `meilisearch_index_docs_count{index="$Index"}` , do you think there is a better way to put it?

the last action failed due to the single `=` before the 0.

Comment (samber) [reply _data/rules.yml:853]: I'm pretty sure that the `$Index` is a variable injected by Grafana.

If you want to match every indices, I suppose the `meilisearch_index_docs_count == 0` would work.

(I cannot test)

Comment (nohant) [reply _data/rules.yml]: uuuuh, youre right @samber 

We se that variable and use it in explore, but without the label will it be more general and clean!
thanks, i wanted to be more precise but that is good enough for a standard alert. :3.

**Review by samber — APPROVED** [review]

**Comment by samber** [comment]
let's merge ;)

**Pull request merged**